includes/rest/class-interaction-controller.php

```diff
@@ -46,28 +51,19 @@ public static function register_routes() {
 	}
 
 	/**
-	 * Handle GET request.
+	 * Retrieves the interaction URL for a given URI.
 	 *
 	 * @param \WP_REST_Request $request The request object.
 	 *
-	 * @return WP_REST_Response Redirect to the editor or die.
+	 * @return \WP_REST_Response|\WP_Error Response object on success, or WP_Error object on failure.
 	 */
-	public static function get( $request ) {
+	public function get_item( $request ) {
 		$uri          = $request->get_param( 'uri' );
 		$redirect_url = null;
-		$object       = Http::get_remote_object( $uri );
+		$object       = \Activitypub\Http::get_remote_object( $uri );
```

Comment:
let's stick with using `use` for Namespaced classes. this is easier to read and consistent with the rest of the code.

Reply:
Fixed in fbbd9d5.